hostinfo/sshd.lua

```diff
@@ -0,0 +1,76 @@
+--[[
+Copyright 2014 Rackspace
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--]]
+local HostInfo = require('./base').HostInfo
+
+local string = require('string')
+local fmt = require('string').format
+local table = require('table')
+local os = require('os')
+local spawn = require('childprocess').spawn
+
+--[[ SSHd Variables ]]--
+local Info = HostInfo:extend()
+function Info:initialize()
+  HostInfo.initialize(self)
+end
+
+function Info:run(callback)
+  if os.type() ~= 'Linux' and os.type() ~= 'Darwin' then
```

Comment:
isn't it really just, is `sshd` in $PATH, not an OS specific check?  eg, this should work on FreeBSD out of the box AFAIK.